contributing.md

```diff
@@ -98,7 +138,21 @@ Note: After each change in your code, you have to run:
 $ docker build -t pydp:test .
 ```
 
-## Coding Standards
+# Contributing
+
+### Beginner Issues
+
+If you are new to the project and want to get into the code, we recommend picking an issue with the label "good first issue". These issues should only require general programming knowledge and little to none insights into the project.
+
+### Issue Allocation
+
+Each issue someone is currently working on should have an assignee. If you want to contribute to an issue someone else is already working on please make sure to get in contact with that person via slack or github and organize yourself.
+
+If you want to work on an open issue, please post a comment telling that you will work on that issue, we will assign you as the assignee then.
```

Comment:
```suggestion
If you want to work on an open issue, please post a comment informing us of your thoughts regarding the issue, so that we can officially assign you for it.
```